tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -176,6 +219,9 @@ def graph_impl(self, run, tag, is_conceptual, limit_attr_size=None, large_attrs_
 
   def run_metadata_impl(self, run, tag):
     """Result of the form `(body, mime_type)`, or `None` if no data exists."""
+    if self._data_provider:
+      # TODO(davidsoergel, wchargin): Consider plumbing run metadata through data providers.
+      return (None, None)
```

Comment:
Should this `(None, None)` be just `None`, like the other error cases?
If I read this correctly, it’ll 500 when hit, on line 296 below.

Reply:
Oops yes.  Just had to defer unpacking the tuple in the associated test, which is more correct anyway.